### PR TITLE
fix(profile): show first nft image as metadata

### PR DESCRIPTION
**Pull request by roiLeo**
#465 related
Capture d’écran 2021-06-22 à 9 02 40 AM

### CODE REVIEW

**Comment by yangwao** [comment]
Seems doesn't work in tg or Twitter 👀
IMG_20210622_091158
IMG_20210622_091217

File: src/views/Profile.vue [hunk 1]
```diff
@@ -166,6 +168,11 @@ export default class Profile extends Vue {
     return this.currentCollectionPage * this.first - this.first;
   }
 
+  get defaultNFTImage() {
+    const url = new URL(window.location.href);
+    return `${url.protocol}${url.hostname}/img/kodadot_logo_v1_transparent_400px.56bb186b.png`;
```
Comment (yangwao) [inline src/views/Profile.vue:173]: `.56bb186b.` in file name is probably just bundle hash, you want to remove it :)

**Review by yangwao — CHANGES_REQUESTED** [review]
you need to rename that default file as you mentioned hash bundle there

**Comment by yangwao** [comment]
Still doesn't render... 
Maybe try to add these, I sense it's conditional for that
`twitter:title`
`og:title`

**Comment by roiLeo** [comment]
I think the card is not showing because we don't provide 'description' and 'title' for twitter and open graph. 
I will try to set with some data. Should I rely on the first nft ?

**Comment by yangwao** [comment]
Okay, that would be awesome, yes try that one.
Meanwhile I'm testing it on other PR til lit build with some generic ones https://github.com/kodadot/nft-gallery/pull/493
Will know in few seconds :)

**Comment by yangwao** [comment]
Oh hint, to mitigate cache, it's fine to add some parameter, like ?abc to end of URL while testing rendering on crawlers, as some cache it hard, even for 3days+

**Comment by yangwao** [comment]
This one seems working tho! 
https://deploy-preview-493--nftkodadot.netlify.app/rmrk/u/CxQJNQeowRUJBgUHQpspudWy4BU6Rk2u93yDZEKe6GzcLDe?abd

image

**Comment by yangwao** [comment]
Well now is the question, why doesn't fetch at least the default picture if I don't see any other? 
image

**Comment by roiLeo** [comment]
My SEO plugin show the right picture and description but on twitter card it show the default data and I don't know why. 
Capture d’écran 2021-06-22 à 4 34 03 PM
Screenshot 2021-06-22 at 16-36-59 Card Validator Twitter Developers

**Comment by yangwao** [comment]
I mean, let's push it to production.
You are welcome to show on our discord https://discord.gg/35hzy2dXXh
Let's reach me there, @yangwao , so I can process some small bounty for your work :)
You will need to have KSM address

**Comment by yangwao** [comment]
For now, I'll merge it as it's at least something and you're welcome to continue cracking why doesn't work:)

**Pull request merged**